src/db.js

```
@@ -0,0 +1,19 @@
+'use strict';
+
+const { Sequelize } = require('sequelize');
+const utils = require('util');
+
+require('dotenv').config();
+global.TextEncoder = utils.TextEncoder;
+
+const { DB_HOST, DB_PORT, DB_USER, DB_PASSWORD, DB_NAME } = process.env;
+const sequelize = new Sequelize({
+  database: DB_NAME || 'postgres',
```

Consider validating required DB environment variables (`DB_HOST`, `DB_PORT`, `DB_USER`, `DB_NAME`, `DB_PASSWORD`) at startup and fail fast with a clear error if any required variable is missing. This prevents the application from starting with unintended defaults and matches the project's robustness goals.